docs/installation.adoc

```diff
@@ -21,7 +21,7 @@ First add the `+helm+` repository, if you haven't already:
 
 [source, bash]
 ----
-helm repo add flink-operator-repo https://downloads.apache.org/flink/flink-kubernetes-operator-1.10.0/
+helm repo add flink-operator-repo https://downloads.apache.org/flink/flink-kubernetes-operator-1.11.0/
```

Comment:
So this is embarassing! I forgot to update the docs for the 0.2.0 release to point to the new operator version! I will have to add some instructions to the release process to remind people to do that.